pkgs/applications/editors/amp/default.nix

```diff
@@ -12,7 +12,14 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-xNadwz2agPbxvgUqrUf1+KsWTmeNh8hJIWcNwTzzM/M=";
   };
 
-  cargoHash = "sha256-4c72l3R9OyxvslKC4RrIu/Ka3grGxIUCY6iF/NHS5X8=";
+  cargoPatches = [
+    (fetchpatch2 {
+      url = "https://github.com/jmacdonald/amp/commit/4ce866de7a2e1613951002ff61563a80e19a5c0c.patch";
```

Comment:
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

We should include the patch file if it's not upstream, because the URL won't be stable.